lib/dataSources.js

```diff
@@ -50,9 +51,12 @@ const update = (conn, name, options) => {
   }).then(httpBody);
 };
 
-const remove = (conn, name) => {
```

Comment:
can you also add the docs to the `index.d.ts` file and run `yarn docs` to update the readme?